packages/widget/src/index.js

```diff
@@ -125,3 +126,9 @@ export async function render(view) {
     reset();
   };
 }
+
+async function instantiateSpec(spec) {
+  const ast = parseSpec(spec);
+  const dom = await astToDOM(ast);
```

Comment:
@domoritz The new `astToDOM()` method returns an object that includes the DOM `element` but also a `Map` of name-Param pairs (including all Param and Selection instances in the spec). This might be handy for better returning Param and Selection values from the browser back to the Jupyter kernel. Feel free to push updates to the widget package on this branch if you want to incorporate this!

Reply:
Awesome. Done in f03de439ef9e37.